### DISCUSSION TITLE
Multiple Objects

### DISCUSSION THREAD

**Discussion by begumbeyza** [google/model-viewer#3234] [Q&A]
Hi,I have a question.I want to use multipe object at the same time.I want to use both enviroment ,SceneViewer,QuickLook and placing objects in different places in the environment.How should I do?Is that possible?

Thank you.

**Comment by samaneh-kazemi** [comment]
Hi @begumbeyza, thanks for reaching out. This is currently not supported due to complex UX requirements.

**Reply by Melis1294** [reply]
Are you planning to develop this feature? Is it in development? Thank you!

**Reply by elalish** [reply]
It's planned, but moving slowly. Also, Scene Viewer and Quick Look have no way to move objects separately in AR and are very unlikely to, so we would only be able to support this feature in our WebXR AR mode.
